apps/react-18-tests-v8/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-18-tests-v8",
   "description": "React 18 test application",
-  "version": "1.0.0",
+  "version": "8.0.0",
```

Comment:
this is needed as tooling checks version and tweaks its behaviours based on that ( cypress )